auth/server/src/main/java/com/seagates3/controller/IAMController.java

```diff
@@ -132,8 +132,9 @@ ServerResponse serve(FullHttpRequest httpRequest,
         LOGGER.error("Incorrect signature. Request not authenticated");
         return serverResponse;
       }
-    } else if (requestAction.equals("UpdateAccountLoginProfile")) {
-      LOGGER.debug("Parsing UpdateAccountLoginProfile request" +
+    } else if (requestAction.equals("UpdateAccountLoginProfile") ||
+               requestAction.equals("DeleteAccount")) {
+      LOGGER.debug("Parsing UpdateAccountLoginProfile/DeleteAccount request" +
```

Comment:
This debug statement isnt use full as we already log request in debug level

Reply:
removed.